website/docs/Examples/Integrate - OpenAI.md

```diff
@@ -0,0 +1,158 @@
+FLAML has integrated the OpenAI's completion API. 
```

Comment:
This will be the first sentence a reader of the paper will see. It could be first time they visit FLAML. Think of what to say here to keep them interested in reading the rest.